src/main/java/com/derongan/minecraft/deeperworld/ecs/DeeperWorldSection.kt

```diff
@@ -0,0 +1,11 @@
+package com.derongan.minecraft.deeperworld.ecs
+
+import com.mineinabyss.deeperworld.world.section.SectionKey
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+@SerialName("deeperworld:section")
+//TODO add other features of a section.
+data class DeeperWorldSection(val name: SectionKey)
```

Comment:
In Geary I chose to give these classes simple names (Section) and use a `typealias DeeperWorldSection = Section` for others to use